code/game/objects/items/storage/smartpack.dm

```diff
@@ -14,14 +20,15 @@
 	actions_types = list(/datum/action/item_action/toggle)
 	xeno_types = null
 
-	var/show_exoskeleton = FALSE
+	var/show_exoskeleton = TRUE
 
 	var/flashlight_cooldown = 0 //Cooldown for toggling the light
 	var/light_state = FALSE //Is the light on or off
 
 	var/battery_charge = SMARTPACK_MAX_POWER_STORED //How much power are we storing
 	var/activated_form = FALSE
 	var/immobile_form = FALSE
+	var/repair_form = FALSE
```

Comment:
you could make all these into flags/a single var with defines... i wont force you... but you could...

Reply:
nyoooo